scripts/update-reg-repo.sh

```diff
@@ -0,0 +1,73 @@
+#!/usr/bin/env bash
+
+# Write output to error output stream.
+echo_stderr() {
+  echo -e "${1}" >&2
+}
+
+die()
+{
+  local _return="${2:-1}"
+  echo_stderr "$1"
+  exit "${_return}"
+}
```

Comment:
these could be a source as well, and maybe rename them to match what we have on openebs

Reply:
Openebs seems to have these same names it seems? Can you point me to the one you are mentioning?